changelog.txt

```diff
@@ -1,501 +1,502 @@
-* Workaround for servers that send additional characters on ajax responses
-
-2017-05-02 - version 1.2.2
-* Fix issues with non-english variant labels
-* Support gallery image urls
-
-2017-05-02 - version 1.2.1
-* Use wc_get_cart_url to support custom cart URLs
-* Fix logging bug for users without manage_woocommerce attempting to set up
-
-2017-04-27 - version 1.2.0
-* Allow regular admins with manage_woocommerce permission to use plugin
-* Lock settings during product sync
-* Do not allow product sync during an existing product sync
-* Validate catalog ID before product sync
-* Fix issue with resync products button
-* Fix issue with plugin breaking product search
-
-2017-04-25 - version 1.1.0
-* Support duplicate SKUs via better retailer ID logic
-* Add "resync products" button
-* Update FB logging to support objects as well as strings
-* Add support for regular and sale pricing
-
-2017-04-20 - version 1.0.3
-* Remove delete button (use Advanced Settings tab instead)
-* Cleaned up admin messages, prepended them with "Facebook for WooCommerce"
-
-2017-04-13 - version 1.0.2
-* Remove Save Settings button when not in debug mode
-
-2017-04-11 - version 1.0.1
-* Prevent blank item descriptions
-* Update copy on setup page
-* Sanitize settings before sending via fblog
-* Show better warnings for duplicate SKUs
-
-2017-04-10 - version 1.0.0
- * First release! Woo!
-
-2017-04-09 - version 0.8.0
- * Include status messages during background sync
- * Fix several PHP warnings and various WooCommerce 3.0 compatibility issues
-
-2017-04-07 - version 0.7.2
- * Fix issue where products were not being created due to blank descriptions
-
-2017-04-04 - version 0.7.1
- * Clean up output for checkboxes via printf
- * Bug fixes and performance improvements
-
-2017-04-03 - version 0.7.0
- * Add new product visibility checkbox toggle for Facebook Shops on Product list
- * Official rename: Facebook for WooCommerce
-
-2017-03-30 - version 0.6.0
- * Revert back to minor revisions for versioning
- * Add ability to reset all settings to start over
-
-2017-03-20 - version 0.0.5
- * Fix "Get Started" button bug
- * Fix issue preventing "Use Advanced Matching" setting from properly saving
- * Various security fixes
-
-2017-03-01 - version 0.0.4
-
- * Add Facebook metabox on product page with products' FB ID and a Visibility toggle (publish/unpublish).
-
-2017-02-15 - version 0.0.3
-
- * Add a change log!
- * Modify plugin to check for both local and network installs of WooCommerce.
- * Disable plugin hooks that were triggering before setup was complete.
- * Escape HTML tags in product descriptions.
- * Use attribute label name instead of first-letter-capitalized slug.
-
-2017-02-02 - version 0.0.2
-
- * Fix breaking when debug mode is disabled (null check in facebook-settings.js)
+*** Facebook for WooCommerce Changelog ***
+
+2020.nn.nn - version 2.0.4
+ * Fix - Prevent an error during the feed generation when variable products are still using deleted terms
```

Comment:
Not sure why it's considering I've changed the whole file since only line 4 was changed.

Probably PhpStorm messed up with the line breaks or something. If that's a problem, I can remove this file from this PR and add the changelog entry directly into `release-2.0.4`.

Reply:
Maybe it's the line break is different in your PHPstorm settings ... it happens to me also

Reply:
@N-Molham Fixed in https://github.com/facebookincubator/facebook-for-woocommerce/pull/1624/commits/4f9267ba2d24f4888834efceea4219843418b4ee. Thanks for reviewing.